core/src/stage.cpp

```diff
@@ -865,7 +865,6 @@ template bool ConnectingPrivate::hasPendingOpposites<Interface::BACKWARD>(const
                                                                           const InterfaceState* start) const;
 
 bool ConnectingPrivate::canCompute() const {
-	ROS_DEBUG_STREAM("canCompute " << name() << ": " << pendingPairsPrinter());
```

Comment:
Can we keep this in namespace `ConnectingPairs` or similar? At least keep the commented out line in please.

Reply:
Sure. Done.